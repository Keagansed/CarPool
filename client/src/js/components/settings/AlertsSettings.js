import React, { Component } from 'react';

class AlertsSettings  extends Component {
    render(){
        return(
            <div className="scroll-vert txt-white">
                Alerts Settings
            </div>
        );
    }
}

export default AlertsSettings;