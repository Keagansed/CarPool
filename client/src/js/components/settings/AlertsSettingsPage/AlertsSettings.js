import React, { Component } from 'react';

class AlertsSettings  extends Component {
    render(){
        return(
            <div className="scroll-vert">
                Alerts Settings
            </div>
        );
    }
}

export default AlertsSettings;