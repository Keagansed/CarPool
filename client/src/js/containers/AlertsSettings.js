// File Type: Component

import React, { Component } from 'react';

/*
* Purpose: Container component that will hold all relevant 'Alert Settings' components
*/
class AlertsSettings extends Component {
    render() {
        return(
            <div className="scroll-vert txt-white">
                Alerts Settings
            </div>
        );
    }
}

export default AlertsSettings;