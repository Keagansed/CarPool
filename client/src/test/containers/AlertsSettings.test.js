import React from 'react';
import AlertsSettings from '../../js/containers/AlertsSettings';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

describe('AlertsSettings Component', () => {

    it ('captures snapshot', () => {
        const renderedValue = renderer.create(<AlertsSettings />).toJSON()
        expect(renderedValue).toMatchSnapshot();
    });

    it ('renders correctly', () => {
        let wrapper = shallow(<AlertsSettings />)
        expect(wrapper.length).toEqual(1);
    });
});