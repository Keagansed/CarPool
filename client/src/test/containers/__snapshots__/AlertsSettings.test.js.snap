// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertsSettings Component captures snapshot 1`] = `
<div
  className="scroll-vert txt-white"
>
  Alerts Settings
</div>
`;
